src/pages/coalition-partners.tsx

```diff
@@ -14,32 +15,31 @@ import MOREImage from '../../static/Images/Partners/MORE.jpg'
 export default function CoalitionPartners(): JSX.Element {
   return (
     <Layout title='Coalition Partners | Internet for All NYC'>
-      <Heading as='h1' fontSize={24} fontWeight='normal' textAlign='center'>
-        We are a growing coalition of organizations fighting for municipal broadband in New York City.
-      </Heading>
-      
-      <Wrap spacing='100px' justify='center' align='center' m={16}>
-        <WrapItem>
+      <VStack spacing={['64px', '96px']}>
+        <Heading fontSize={24} fontWeight={500} textAlign='center'>
+          We are a growing coalition of organizations fighting for municipal broadband in New York City.
+        </Heading>
+        
+        <Wrap spacing={['40px', '96px']} justify='center'>
           <Link href='https://techaction.nyc'>
-            <Image src={TechActionImage} h={160} />
+            <Image src={TechActionImage} boxSize={[108, 144]} />
           </Link>
-        </WrapItem>
-        <WrapItem>
           <Link href='https://local3ibew.org'>
-            <Image src={IBEWImage} h={144} />
+            <Image src={IBEWImage} boxSize={[108, 144]} />
           </Link>
-        </WrapItem>
-        <WrapItem>
           <Link href='https://www.vocal-ny.org'>
-            <Image src={VocalNYImage} h={144} />
+            <Image src={VocalNYImage} boxSize={[108, 144]} />
           </Link>
-        </WrapItem>
-        <WrapItem>
           <Link href='https://morecaucusnyc.org'>
-            <Image src={MOREImage} h={144} />
+            <Image src={MOREImage} boxSize={[108, 144]} />
```

Comment:
Made the MORE logo a square because all the other logos are and it's easier to display them in a responsive format when they're all the same size